src/lib/mina_base/parties.ml

```diff
@@ -0,0 +1,140 @@
+open Core
+
+[%%versioned
+module Stable = struct
+  module V1 = struct
+    type t =
+      { parties: Party.Signed.Stable.V1.t * Party.Stable.V1.t list
```

Comment:
Nit: it might be more self-documenting if this is split into `fee_payer` and `other_parties`, especially since we'd like it to be obvious that the party's entire committed amount will be transferred to the block producer if the remaining parties' updates fail.